### PR TITLE
766 pivot operation form

### PR DESCRIPTION
**Pull request by Ishankoradia**

### CODE REVIEW

File: src/components/UI/GridTable/GridCheckBox.tsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+      <TableHead sx={castGridStyles.headerItem}>
+        <TableRow>
+          <TableCell
+            sx={{
+              padding: '12px 10px',
+              width: `100%`,
+              borderRight: '1px solid #E8E8E8',
+              display: 'flex',
+              alignItems: 'center',
+            }}
```
Comment (mdshamoon) [inline src/components/UI/GridTable/GridCheckBox.tsx:53]: The table is common in GridTable and GridCheckbox. Can we extend the GridTable component to create GridCheckbox. This way if some style changes are needed we don't have to update in two places

**Review by mdshamoon — CHANGES_REQUESTED** [review]
Functional feedbacks:

1. Pressing enter should add a new row
image

2. Additional left padding on the checkbox
image

File: src/components/UI/GridTable/GridCheckBox.tsx (hunk 1 above)
Comment (Ishankoradia) [reply src/components/UI/GridTable/GridCheckBox.tsx:53]: yes agreed. Let me think about it and make the necessary changes.

Comment (Ishankoradia) [reply src/components/UI/GridTable/GridCheckBox.tsx]: Done. Have re-used the gridtable

**Comment by Ishankoradia** [comment]
> Functional feedbacks:
> 
> 1. Pressing enter should add a new row
> 
> image
> 2. Need additional left padding on the checkbox
> 
> image

Made the change

**Review by mdshamoon — APPROVED** [review]

**Pull request merged**